blueprint/index.md

```diff
@@ -84,9 +84,29 @@ The integration provides the following examples of Architect flows:
 
 ### HRIS-Get-Agents flow
 
-This HRIS provides a list of all agent records, including their corresponding IDs/keys and email addresses. Genesys Cloud workforce management requires this data to retrieve time-off balances and insert time-off requests for an agent.
+The primary purpose of this workflow is to retrieve the list of agents configured in external HRIS system and that information is used in Genesys's HRIS agent synchronization process that is scheduled to run once every 24 hours.
```

Comment:
Is there secondary purpose. If no, let's remove "primary". 
I'd say the purpose is to provide unique ids of agents in external HRIS system together with their emails, according to which these agents can be mapped to the user records in Genesys Cloud.

Reply:
will do